modules/wallet/scan.go

```diff
@@ -11,38 +11,38 @@ import (
 
 const scanMultiplier = 4 // how many more keys to generate after each scan iteration
 
-// numInitialKeys is the number of keys generated by the seedScanner before
-// scanning the blockchain for the first time.
-var numInitialKeys = func() uint64 {
+var errMaxKeys = fmt.Errorf("refused to generate more than %v keys from seed", maxScanKeys)
+
+// maxScanKeys is the number of maximum number of keys the seedScanner will
+// generate before giving up.
+var maxScanKeys = func() uint64 {
 	switch build.Release {
 	case "dev":
-		return 10e3
-	case "standard":
 		return 1e6
+	case "standard":
+		return 100e6
 	case "testing":
-		return 1e3
+		return 100e3
 	default:
 		panic("unrecognized build.Release")
 	}
 }()
 
-// maxScanKeys is the number of maximum number of keys the seedScanner will
-// generate before giving up.
-var maxScanKeys = func() uint64 {
+// numInitialKeys is the number of keys generated by the seedScanner before
+// scanning the blockchain for the first time.
+var numInitialKeys = func() uint64 {
 	switch build.Release {
 	case "dev":
-		return 1e6
+		return 10e3
 	case "standard":
-		return 100e6
+		return 1e6
 	case "testing":
-		return 100e3
+		return 1e3
```

Comment:
this freaked me out a little but I think it's just a weird diff

Reply:
I swapped large expression having similarities so diff is really weird.